advanced/helm/integrator-with-analytics/integrator-with-analytics/auth.json

```diff
@@ -1,10 +1,10 @@
 {
```

Comment:
@Savidude I think we can move the Helm resources to the parent folder from the sub folder with the same pattern name as we did in the case of Helm resources for WSO2 IAM deployment patterns.

Reply:
Shall we do this once al of the helm resources have been updated with the single chart deployment setup?

Reply:
+1